streams/src/main/java/org/apache/kafka/streams/kstream/internals/KGroupedStreamImpl.java

```diff
@@ -350,6 +356,17 @@ private void determineIsQueryable(final String queryableStoreName) {
                                          repartitionRequired);
     }
 
+    @Override
+    public SessionWindowedKStream<K, V> windowedBy(final SessionWindows windows) {
+        return new SessionWindowedKStreamImpl<>(windows,
+                                                builder,
+                                                sourceNodes,
+                                                name,
+                                                keySerde,
+                                                valSerde,
+                                                aggregateBuilder);
+    }
+
```

Comment:
For line 371 below: should we do the same as for the deprecated `aggregate` to use `windowedBy` internally for this function?